conf/core/NetworkAlgorithm.conf

```diff
@@ -1,8 +1,12 @@
 {
   "conflate.match.highway.classifier": "hoot::HighwayExpertClassifier",
   "conflate.pre.ops": "$(conflate.pre.ops);hoot::RubberSheet",
-  "match.creators": "hoot::BuildingMatchCreator;hoot::NetworkMatchCreator;hoot::PoiPolygonMatchCreator;hoot::ScriptMatchCreator,Area.js;hoot::ScriptMatchCreator,LinearWaterway.js;hoot::ScriptMatchCreator,PoiGeneric.js;hoot::ScriptMatchCreator,Railway.js;hoot::ScriptMatchCreator,PowerLine.js",
-  "merger.creators": "hoot::BuildingMergerCreator;hoot::NetworkMergerCreator;hoot::PoiPolygonMergerCreator;hoot::ScriptMergerCreator;hoot::ScriptMergerCreator;hoot::ScriptMergerCreator;hoot::ScriptMergerCreator;hoot::ScriptMergerCreator",
+  "#": "Notice that this match creator ordering is slightly different than the one used by Unifying. Making the ordering the same as",
```

Comment:
Its very interesting to note that this ordering is different than that used with Unifying alg and if you change it to match you get a bunch of weird test output changes. Opened #3722 to look into it.